### ISSUE TITLE
Integrities are concatenated

### ISSUE THREAD

**Issue by Shakeskeyboarde** [JamieMason/shrinkpack#96]
## Description

When run multiple times, the lock file changes every time, because the integrity strings are concatenated to the old integrity value.

## Suggested Solution

Replace the integrity value instead of adding to it.

## Help Needed

N/A

**Comment by JamieMason** [comment]
Thanks for reporting, will take a look.

**Comment by JamieMason** [comment]
Do you have a fix for this in your rewrite @Shakeskeyboarde? No worries if not but may save me some time if so.

**Comment by Shakeskeyboarde** [comment]
I'm not sure I have a fix exactly, but here's my lock file handler: https://github.com/Shakeskeyboarde/npm-offline-mirror/blob/main/src/lock.ts

**Comment by alltom** [comment]
Besides the noisy diffs, does this cause any problems? I noticed this in a new project, where it's my first time using shrinkpack.

**Comment by JamieMason** [comment]
I don't think it causes problems no, it just may have more hashes in that it needs to.
